docs/how-to-guides.md

```diff
@@ -32,6 +32,199 @@ Everything about test data generation and its adjustable parameters can
 be found in the [Generating a Test Dataset with niceML](
 generate-data.md#step-3-customizing-data-generation-optional)
 
+## How to write my own pipeline
+NiceML provides configurable, re-usable machine learning 
+pipelines. NiceML comes with pre-configured pipelines for object detection, 
+semantic semgmentation, regression, classification and others. But what if 
+your problem is not covered (yet) by the pipeline portfolio niceML offers?
+This guide will walk you through how to set up your first custom pipeline.
```

Comment:
Suggestion: This guide will walk you through the setup of your first custom pipeline.